R/grade_conditions.R

```diff
@@ -32,25 +36,8 @@
 #' @seealso `test`
 #' @export
 #' @examples
-#' \dontrun{grading_demo()}
-#'
-#' example_function <- function(x){
```

Comment:
didn't want to keep these?

Reply:
uhh... correct, I missed this during rebase

Reply:
Ok. Keep them off or add them back if you’d like